docs/concepts/credentials/0_overview.md

```diff
@@ -0,0 +1,24 @@
+---
+id: overview
+title: Credentials - Overview
+---
+
+**Credentials** are attested claims which are owned by a **Claimer**, attested by an **Attester** and can be verified by a **Verifier**.
+
+To get a Credential a Claimer needs to go through following process:
+1. Find a **CType**, a Claim should be based on (Can be advertised by the potential Attester and Verifier)
+2. Make a **Claim**
+3. Potentially request and receive **Terms** and agree on a **Quote** with an Attester
+4. **Request an Attestation**
+5. Wait for the Claim to be **attested** on-chain by the Attester
+
+To verify the Credential a Claimer can send a presentation of it to a Verifier
+
+1. The Verifier may request a **Credential** as the first step, providing a nonce for security reasons
```

Comment:
Same here, probably does not add anything valuable considering this is an introductory section, does it?

```suggestion
1. The Verifier may request a **Credential** as the first step
```